src/components/tools/swot-analysis/steps/SWOTClassifyAlternativeActions/NormalClassifying/NormalClassifying.tsx

```diff
@@ -1,177 +1,47 @@
-                                    </InputGroup>
-                                </Accordion.Header>
-                                <Accordion.Body>
-                                    <ClassifyingCardList
-                                        disabled={this.props.disabled}
-                                        actions={classification.actions}
-                                        onOpenClassificationModalClick={this.onOpenClassificationModalClick}
-                                    />
-                                </Accordion.Body>
-                            </Accordion.Item>
-                        );
-                    })}
-                </Accordion>
-
-                {(this.props.disabled && classifications.length <= 0) && (
-                    <span>
-                        Keine Klassifikationen vorhanden...
-                    </span>
-                )}
-
-                {(!this.props.disabled && (
-                        SWOTClassifyAlternativeActions.maxClassifications
-                        > classifications.length)
-                ) && (
-                    <Button onClick={this.props.classificationController.addClassification}
-                            className={"addClassification"}>
-                        <FAE icon={faPlus} color={"white"}/>
-                    </Button>
-                )}
-
-                <hr/>
-
-                <div className={"actionCards"}>
-                    {actions.filter(value => !value.alreadyAdded).map((action, index) => {
-                        return (
-                            <ClassifyingCard
-                                key={"classifying-card-" + action.name + "-" + index}
-                                action={action}
-                                disabled={this.props.disabled}
-                                onChangeClick={this.onOpenClassificationModalClick}
-                            />
-                        );
-                    })}
-
-                    {(!actions.some((v) => !v.alreadyAdded)) && (
-                        <span>Alle zugeordnet!</span>
-                    )}
-                </div>
-
-
-                <SelectClassificationModal
-                    open={this.state.openClassificationModal}
-                    action={this.state.lastSelectedAction ?? undefined}
+                <ClassificationAccordions
                     classifications={classifications}
-                    onSelect={this.changeClassification}
-                    onClose={this.closeClassificationModal}
+                    disabled={this.props.disabled}
+                    classificationController={this.props.classificationController}
+                    onClassificationClick={() => {
+                    }}
+                    actions={this.props.actions}
+                    cardElement={
+                        <ClassifyingMenuCard
+                            classifications={classifications.filter((c) => c.name.length > 0)}
```

Comment:
I think it would be better to set the disabled and click props here. So other command in ClassifyingCardList.